main.py

```diff
@@ -4,7 +4,7 @@
 from ia32doc.processors.c_processor import DocCProcessor
 from ia32doc.options import DocProcessorOptions
 
-ALL = False
+ALL = True
```

Comment:
You probably don't want to commit this.

Reply:
I found this is useful because when you update YML files, you want to update all header flavors. I did not notice any downside too. Can you explain your concern?

Reply:
My concern is that the main() function with it's argument parser gets totally ignored. :) It would probably be better to add another option to the argparse (`-a` ?) to signalize "all".

Either that, or make it default behavior to run "ALL" when no arguments are passed.

Reply:
Makes sense. I pushed the change to make the ALL behaviour by default when no file is specified. 

Another push was to fix a compatibility issue with the newer YAML lib.
https://github.com/yaml/pyyaml/blob/master/CHANGES
```
* https://github.com/yaml/pyyaml/pull/561 -- always require `Loader` arg to `yaml.load()`
```

Reply:
Looks great, thanks!